test/openal_extensions.c

```diff
@@ -0,0 +1,71 @@
+// So far just does some sanity checks for the available extensions.
+//
+// In the future, could also try to enable different extensions and test for
+// output correctness.
+
+#include <stdio.h>
+#include <stdlib.h>
+#ifdef __EMSCRIPTEN__
+#include <emscripten.h>
```

Comment:
Is this include needed?

Reply:
Unsure, copied from openal_capture_sanity.c, since I saw it in a couple of files. I can remove and see if the test passes.